akka-actor/src/main/scala/akka/util/ConstantFun.scala

```diff
@@ -1,7 +1,7 @@
 /**
  * Copyright (C) 2015-2017 Lightbend Inc. <http://www.lightbend.com>
  */
-package akka.stream.impl
+package akka.util
```

Comment:
Then I'd still propose to copy it over and deprecate old ones.
It's very noticeable changes in quite a few hotpaths and having a place for them is a nice thing to have.